content/blog/urbit-creator-daos.md

```diff
@@ -0,0 +1,74 @@
+
+# An Owned Space
+
+These use cases help with DAO flexibility, usability, and decentralized automation. But another important reason to build a DAO on Urbit is that you’re building your network on P2P “real estate.” On Urbit, both creators and their communities own the address space and every action taken in it—it can’t be canceled or deleted or made inoperable, a far cry from other platforms where you can be suspended for arbitrary Terms of Service violations. 
+
+“The main positive is you and your community getting to own the real estate that you operate on,” says Murphy. No other app, even dapps like Agora or Snapshot, allows for a genuinely P2P offchain operating environment. Your Urbit ID is self-owned and portable, and you build your community P2P. Your creations are yours and you can continue building them without corporate middlemen ripping your canvas out from under you, or changing your tools or access without notice.
+
+![Image](https://storage.googleapis.com/media.urbit.org/blog/otherlifegroup.png)
+<div style="font-size:13px; text-align: center; color: #999">~hatryx-lastud/other-life</div>
+
+Furthermore, the more work you put into building your Creator DAO on Urbit, the more valuable the Urbit address space becomes. When you use Facebook, you’re relegated to the role of a content serf. The more you produce, the more Facebook grows in value. On Urbit, the more your produce, the more Urbit grows in value—and you are an owner of Urbit. 
+
+![Image](https://storage.googleapis.com/media.urbit.org/blog/justinmurphy.jpg)
+<div style="font-size:13px; text-align: center; color: #999">Murphy already offers his members access to the Other Life urbit group.</div>
+
+Finally, Murphy cites Urbit’s durability.
+
+> The second most important Urbit trait after ownership is durability. The creator economy tools change all the time. Every time you try to move your people from one new app to another app, you lose a lot, right? You want to be an influencer, you want to be an important thinker and speaker. All of a sudden, you’ve got to be on Clubhouse! This means telling the people on your newsletter, telling your Patreon subscribers, telling the people on your podcast, hey, download this cool new app called Clubhouse and find me there. It's going to be awesome. It's so cringe and corny and only a small number of them will even make it over there. Then all of a sudden, no one's on Clubhouse and no one wants to use Clubhouse anymore. This is a major problem for people navigating the creator economy. I think one of the big selling points of Urbit is—this is the one place that I’ll always be. I’ve been working on my Urbit for over 10 years. This thing is going to be the nuclear bunker of the internet.
```

Comment:
97% sure jmurphy hasn't been on urbit for a decade. I think I understand the context here as 'in the future he would be able to say something like this' but it's not totally clear if you don't know the broader context of justin on urbit.